src/routes/files/handlers.js

```diff
@@ -0,0 +1,109 @@
+import { v4 as uuidv4 } from 'uuid';
+import * as fs from 'fs';
+import handleErrors from '../../lib/apiErrorHandler';
+import { sequelize, File } from '../../models';
+import s3Uploader from '../../lib/s3Uploader';
+
+const fileType = require('file-type');
+const multiparty = require('multiparty');
+
+const namespace = 'SERVICE:FILES';
+
+const ATTACHMENT = 'ATTACHMENT';
+const RESOURCE = 'RESOURCE';
+
+const logContext = {
+  namespace,
+};
+
+export const createFileMetaData = async (
+  originalFileName,
+  s3FileName,
+  reportId,
+  attachmentType,
+) => {
+  const newFile = {
+    activityReportId: reportId,
+    originalFileName,
+    attachmentType,
+    key: s3FileName,
+    status: 'UPLOADING',
+  };
+  let file;
+  try {
+    await sequelize.transaction(async (transaction) => {
```

Comment:
What's the benefit of wrapping this (and the update call in `updateStatus`) in a transaction?

Reply:
I don't know, I am new to sequalize and found that was how josh had done it elsewhere, so I copied it. I THINK it would be fine not too.